recipes-graphics/gtk+/gtk+3_%.bbappend

```diff
@@ -1,6 +1,6 @@
 DEPENDS:append:imxgpu2d = " virtual/egl"
 
-WAYLAND = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', '', 'wayland', d)}"
+WAYLAND = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', '', 'x11', d)}"
 WAYLANDONLY = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '${WAYLAND}', '', d)}"
```

Comment:
The point of the logic was to disable `x11` support if `wayland` is configured. This change just negates that, so might as well just drop the logic entirely, right? I'm doing a build now to see how this affects us.

Reply:
And why do you consider it a logical error? We've had to do this same logic for other packages because the package when configured for X11 would fail because our GPU drivers no longer support GLX.

Reply:
it was my logic error :)

Reply:
Ah, of course, my logical error reading ;-)

Reply:
> The point of the logic was to disable `x11` support if `wayland` is configured. This change just negates that, so might as well just drop the logic entirely, right? I'm doing a build now to see how this affects us.

The change builds fine for me. Shall we drop this logic entirely?